Libraries/LLM/Models.swift

```diff
@@ -110,6 +110,13 @@ public struct ModelConfiguration {
 }
 
 extension ModelConfiguration {
+    public static let smolLM_135M_4bit = ModelConfiguration(
```

Comment:
I noticed the 4-bit model is pretty low quality. It might be better to default this to the 8bit (or 16bit).

Reply:
Even the 1.7B instruction-tuned model at 8-bit doesn't work well for me. Maybe they didn't do enough fine tuning?